flink-runtime/src/main/java/org/apache/flink/streaming/api/operators/StreamingRuntimeContext.java

```diff
@@ -110,6 +112,8 @@ public StreamingRuntimeContext(
         this.operatorUniqueID = checkNotNull(operatorID).toString();
         this.processingTimeService = processingTimeService;
         this.keyedStateStore = keyedStateStore;
+        // By default, support state v1
+        this.supportKeyedStateApiSet = SupportKeyedStateApiSet.STATE_V1;
```

Comment:
nit: set `supportKeyedStateApiSet=V1` in `setKeyedStateStore(@Nullable KeyedStateStore keyedStateStore)` instead of here?

Reply:
Well I'd rather not. In some test they does not call `setKeyedStateStore` but the `keyedStateStore` is set in constructor, I'd keep those tests unchanged.